File: packages/python/tests/canary_branch_test.py
"""CANARY — do not merge.

Deliberately exercises ONLY the positive branch of ``canary_branch``.
Line coverage is 100%; branch coverage is 50%.
"""

from cachetta.canary_branch import canary_branch


def describe_canary_branch():
    def test_covers_the_positive_side():
        assert canary_branch(5) == 1

File: packages/javascript/src/canary-branch.test.ts
import { describe, it, expect } from 'vitest';
import { canaryBranch } from './canary-branch.js';

/**
 * CANARY — do not merge.
 *
 * Deliberately exercises ONLY the positive branch. Line coverage of
 * canary-branch.ts is 100%; branch coverage is 50%.
 */
describe('canaryBranch (one branch only)', () => {
  it('covers the positive side', () => {
    expect(canaryBranch(5)).toBe(1);
  });
});
